examples/spark/spark-gluster/spark-master-service.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I did not know that. Does it work with our yaml parser? Have you used it with kubectl -f